### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by Kartikgc9** [community/community#119809] [Programming Help]
### Select Topic Area

Question

### Body

Can anyone help in developing a crowd source disaster management project for my college?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Kartikgc9 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

P.S. You should change the title of your post to reflect a project proposal for maximum exposure.

**Comment by ebndev** [comment]
Hi @Kartikgc9, we're happy you're here 😀

However, we currently do not allow job listings or recruitment for projects on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

If you have any specific questions about issues you're facing in the project, I encourage you to post in our [Programming Help 👩‍💻](https://github.com/orgs/community/discussions/categories/programming-help) category!
